testing/object-storage-mock/src/main/java/org/projectnessie/objectstoragemock/AwsChunkedInputStream.java

```diff
@@ -121,6 +125,7 @@ private String readLine() throws IOException {
           throw new EOFException("Unexpected end of metadata line");
         }
         checkArgument(c == 10, "Illegal CR-LF sequence");
+        System.err.println("readLine: " + line);
```

Comment:
`System.err`?

Reply:
You guys should found the same stupid left-over within the same 2 minutes? ;)